charts/tigera-operator/values.yaml

```diff
@@ -1,8 +1,20 @@
+# imagePullSecrets are a special helm field which, when specified, creates a secret
+# containing the pull secret and configures operator's serviceaccount to use it to pull the operator image
```

Comment:
Maybe just "all images deployed by this helm chart and the resulting operator" is a more concise way of saying this?